Name: Improved Edited Video Filenames
Version: 1.4.5-1
Author: Jason Robitaille
Description: Will output edited videos in the format <filename>_<date/time>.mp4  (MIT license)

--- .orig/usr/palm/frameworks/private/metascene.videos/submission/76.5/javascript/assistants/browser-assistant.js
+++ /usr/palm/frameworks/private/metascene.videos/submission/76.5/javascript/assistants/browser-assistant.js
@@ -185,6 +185,17 @@
 		} else {
 			// show details scene
 			params.captured = this.query.captured;
+			var date = new Date();
+			var YYYY = date.getFullYear();
+			var MM = (1+date.getMonth()); if (MM<10) { MM = "0"+MM; }
+			var DD = (0+date.getDate()); if (DD<10) { DD = "0"+DD; }
+			var hh = date.getHours(); if (hh < 10) { hh = "0"+hh; }
+			var mm = date.getMinutes(); if (mm < 10) { mm = "0"+mm; }
+			var ss = date.getSeconds(); if (ss < 10) { ss = "0"+ss; }
+			var fileBase = "_" + YYYY + "-" +  MM + "-" + DD + "-" + hh + mm + ss;
+			params.destination = params.target.substring(0,
+					params.target.lastIndexOf(".")) + fileBase +
+					params.target.substring(params.target.lastIndexOf("."));
 			metascene.Push(this.controller.stageController, exports.Details, params);
 		}
 	}
